hexkit/providers/s3/testutils.py

```diff
@@ -87,6 +86,20 @@ def __init__(self, config: S3Config, storage: S3ObjectStorage):
         """Initialize with config."""
         self.config = config
         self.storage = storage
+        self._buckets: set[str] = set()
+
+    async def empty_buckets(self, buckets_to_exclude: Optional[set[str]] = None):
```

Comment:
Would probably make sense to keep the interface consistent across the different `delete/empty_*` functions - `Optional[Union[str, list[str]]]` - and do the conversion to set in the function.
Made the mistake of passing a string into a set and getting the chars instead of what I actually wanted often enough, i.e. {'Test'} and set('Test') behave differently and I forget often enough and probably other people do too.

Reply:
I changed it to a list and just left it like that inside the function (it's not converted to a set) in [498a676](https://github.com/ghga-de/hexkit/pull/59/commits/498a67631b5d498fde806b88be13e00655e76471)